src/services/keycloakService.ts

```diff
@@ -0,0 +1,122 @@
+import * as WebConfigAPI from '../services/WebConfigService';
+declare global {
+  interface Window {
+    Keycloak?: any;
+  }
+}
+/**
+ * Enum with possible authentication roles in keycloak
```

Comment:
Please use `.` at the end, so `...on roles in keycloak.` See for example `useDataContainer.ts` file as an example. The same on other similar places, I remember having some issues when auto generating docs while missing it.